src/WorldGuard/WorldGuard.php

```diff
@@ -155,12 +196,22 @@ public function sessionizePlayer(Player $player)
         $this->updateRegion($player);
     }
 
+    /**
+     * @param Position $pos
+     *
+     * @return string
+     */
     public function getRegionFromPosition(Position $pos)
```

Comment:
This function can return `Region` too.

Reply:
@Muqsit I get it. Please push the pull request to send another with the changes.

Reply:
You can modify this one itself.

Reply:
@Muqsit I deleted the forked repository.